deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Could you please document here what variables need to be declared? Basically what you did in the PR description...